docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
    - There are 1-week age bins (36-37 weeks, and 37-38 weeks).
    - There are two categories where the age range is 0-24 weeks (all these
      "extremely extreme" preterm births are grouped together).
+
+How is the exposure estimated in GBD 2017
+-----------------------------------------
+   
+To model the joint distribution of exposure of low birth weight and short gestation for each location,
+year, and sex estimated in GBD 2017, three types of information are used:
+
+   - Distribution of gestational age for each location, year, and sex
+   - Distribution of birth weight for each location, year, and sex
+   - Copula family and parameters, specifying correlation between gestational age and birth weight distributions
+
+Exposure modelling strategy in GBD 2017
+---------------------------------------
```

Comment:
I think that the important part to take away here is that GBD models LBWSG exposure in *categories*. I think it would be great to list the categories in this section. I'm not sure that we need so much information about *how* GBD estimates the exposure category distributions, but I think it is fine to include.

Reply:
Another really important part is how we convert from GBD categories to a joint continuous distribution (using Abie's notebook code... assume a uniform distribution within each category... note that this is likely biasing towards overestimating the extreme ends of each category (most babies in 0-500 g category are probably pretty close to 500 g, not equally probable to be 1 gram versus 499 grams).

Reply:
there is a drawn diagram of all the categories, I'm trying to upload it too, but currently not rendering. I'll fiddle with it.

Reply:
And we need to make a to-do about how we could improve this!

Reply:
Oh, gotcha. I just left a comment about this that I think will help!

Reply:
I've deleted most of the details, and left some bare bones description. Let me know if it makes sense as I might not have understood it properly.